### DISCUSSION TITLE
[IMPLEMENTED] Possibility to set Feed rate to same extends as in OctoPrint

### DISCUSSION THREAD

**Discussion by MTrab** [OctoFarm/OctoFarm#251] [Implemented]
In Octofarm Feed rate goes from 50% to 150%, in OctoPrint it's from 10% to 300%
Would be nice to be able to set lower and higher, when loading/unloading filament.

**Comment by NotExpectedYet** [comment]
I didn't realise that had changed it was at one point. Will update that for next release no worries.

**Reply by MTrab** [reply]
You are tha man :D

**Reply by NotExpectedYet** [reply]
Welcome. 

I rarely touch octoprint now a days aha

**Reply by MTrab** [reply]
Me too - actually right now I only touch it for loading/unloading filaments and turning on/off via Enclosure Plugin (until that little issue get fixed)

**Reply by NotExpectedYet** [reply]
Aye I'm planning to sort that for the next release too, confused to why the request doesn't succeed but I'll know more when I install and have a look.

**Comment by NotExpectedYet** [comment]
Can't remeber if I let you know but fixed the issue we're the ui wasn't updated

**Comment by MTrab** [comment]
Confirmed working. A little nice-to-have: Could you add a text field to enter the value by keyboard?

**Reply by davidzwa** [reply]
Quite busy with the UI and server revamp. Could you make an feature request for that on the issues page? That way we can keep track of nice ideas, however big or small.
Please also add a small screenshot where you'd see such a field placed. Thanks in advance!

**Reply by NotExpectedYet** [reply]
I'd prefer feature requests in discussions. I would rather keep issues of stuff were dealing with now if you agree man?

**Reply by davidzwa** [reply]
Yep as elsewhere discussed. Following my old ways of working, not used to discussions. @MTrab sorry for the confusion, we're still figuring out our ways.
